### PR TITLE
chore: Inspectable WebView not applied correctly

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): follow up to #18074

## PR Type

What kind of change does this PR introduce?

- Bugfix

## What is the current behavior?

Inspectable was applied only Uno.UI was built with DEBUG flag

## What is the new behavior?

Apply the flag always

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by MartinZikmund** [comment]
@mergifyio backport release/stable/5.4

File: src/Uno.UI/UI/Xaml/Controls/WebView/Native/iOSmacOS/UnoWKWebView.iOSmacOS.cs [hunk 1]
```diff
@@ -67,7 +67,7 @@ public partial class UnoWKWebView : WKWebView, INativeWebView, IWKScriptMessageH
 			}
 		}
 
-#if __IOS__ && DEBUG
+#if __IOS__
```
Comment (ajpinedam) [inline src/Uno.UI/UI/Xaml/Controls/WebView/Native/iOSmacOS/UnoWKWebView.iOSmacOS.cs:70]: I thought we wanted to get this available only when `DEBUG`

Comment (jeromelaban) [reply src/Uno.UI/UI/Xaml/Controls/WebView/Native/iOSmacOS/UnoWKWebView.iOSmacOS.cs:70]: Right, we do not provide debug builds of our nuget packages. The docs should only mention this flag as debug.

**Review by jeromelaban — APPROVED** [review]

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.4

**Comment by jeromelaban** [comment]
@ajpinedam looks like there's another build error.

**Comment by ajpinedam** [comment]
> @ajpinedam looks like there's another build error.

This Pr was created my @MartinZikmund , do you want me to look a the build error?

File: src/Uno.UI/UI/Xaml/Controls/WebView/Native/iOSmacOS/UnoWKWebView.iOSmacOS.cs (hunk 1 above)
Comment (ajpinedam) [reply src/Uno.UI/UI/Xaml/Controls/WebView/Native/iOSmacOS/UnoWKWebView.iOSmacOS.cs:70]: Right makes sense.

**Comment by ajpinedam** [comment]
I pushed a change to fix the CI.

cc: @MartinZikmund  @jeromelaban

**Review by ajpinedam — APPROVED** [review]

**Pull request merged**